keyboards/nuphy/air75_v2/ansi/readme.md

```diff
@@ -0,0 +1,24 @@
+# NuPhy Air75 V2
+
+*NuPhy Air75 V2 is a standard 84 key keyboard.*
+![NuPhy Air75 V2](https://i.shgcdn.com/cf3f3ed1-3208-4cd5-a6a9-e5f8e6e72d71/-/format/auto/-/preview/3000x3000)
```

Comment:
Please rehost this on Imgur.

Reply:
Thank you, I have already uploaded the image on imgur